app/assets/stylesheets/wigu/active_admin_theme.scss

```diff
@@ -440,7 +440,7 @@ form fieldset.buttons li.cancel a, form fieldset.actions li.cancel a, form.filte
   text-shadow: none;
   background-color: #ffffff;
   @include rounded($skinBorderRadius);
-  padding: 10px 20px;
+  padding: 9px 20px 9px 20px;
```

Comment:
So borders are added to the total height of the button,  Let's take into consideration when setting the padding.